### DISCUSSION TITLE
Filtering out short promos

### DISCUSSION THREAD

**Discussion by AndyZaa** [itsToggle/plex_debrid#616] [Q&A]
On new as-yet unreleased movies that I put in my Plex Watchlist, often I get the advance promos. I've learned to look at the run time to identify the promos - they're usually 2-4 minutes. I delete them and re-enter the movie on the Watchlist. It would be great to not download the promos at all.

I figure I might be able to exclude them by adjusting the 8) size requirement in Options/Settings/Scraper Settings/Version/Edit. That default is set to size requirement >=0.1

Has anyone adjusted this setting to exclude promos that are only a couple of minutes long? My resolution is set to 1080. Any idea what that resolution and time translates to size? Also, any idea of what that number means - is it Megabytes?

Thanks.

**Comment by kent-chan-dev** [comment]
Here is some helpful info for you: https://github.com/itsToggle/plex_debrid/wiki/Version-Guides

To answer your questions, size is in unit GB, so 0.1 is 100mb etc. Can try to set something higher like >=1
Can also find what is common in the promo titles. 
Perhaps exclude (PROMO|DEMO|SAMPLE|INTERNAL)
